### DISCUSSION TITLE
[ FF7 ] Crash report

### DISCUSSION THREAD

**Discussion by Volcannon** [julianxhokaxhiu/FFNx#229] [Q&A]
Hello, I started playing the game with mods and I crashed as soon as I transitioned from the room with the flaming door (just escaped the mako reactor). I will attach my file here, please let me know what went wrong, please?

[crash.zip](https://github.com/julianxhokaxhiu/FFNx/files/6295861/crash.zip)

**Comment by julianxhokaxhiu** [comment]
Hi, which tool are you using to load mods? Are you installing them manually? In any case please approach the mod creator first, as I do work very close with all of them, and all the currently released mods have been heavily tested. If they do crash, it may be a conflict of mods. I'd suggest you joining the supports channels on Discord which may speed up the resolution of your own issue :)

**Reply by Volcannon** [reply]
There was only 2 mods I had to install manually which was the Cetra Project and the Animations by Qhimm because that one in particular was having a error when I tried downloading it from the manager itself (7th Heaven Mod Manager).

Oh, and I usually use the auto sort mods in the categoryt load order button.

**Reply by eve-atum** [reply]
Get rid of animations.  It has been incompatible for a couple of months now, and the links have been intentionally removed from the catalog.
